### DISCUSSION TITLE
Is there any way to do named routing without transition - newest version of stacked (3.1.0-beta.0)

### DISCUSSION THREAD

**Discussion by rr-appadaptive** [FilledStacks/stacked#837] [Q&A]
Although there are various ways to navigate in the newest "stacked" package (navigateTo(), navigateToView()... e. g.), the only stable way in my web-app is to use the named routing (navigateTo(String routeName)). But there I found no way to suppress any transition. (transitionStyle: Transition.noTransition cannot be used). Is there any other way to suppress transition for the stacked_services?

**Comment by FilledStacks** [comment]
@rr-appadaptive we haven't spent a lot of time on the web side of Stacked. 

There's a lot that we need to do there. Is it a good assumptions that having the ability to disable transitions will help you get past this issue?

**Reply by rr-appadaptive** [reply]
Does this mean that the config method of the navigation service also has no influence on the transition? I have tried to change the defaultTransitionStyle to "noTransition" ==> Without success.

**Reply by rr-appadaptive** [reply]
In the meantime I found a way to do the navigation without transition if necessary. If i use the method `_navigationService.navigateWithTransition(<Widget>, transitionStyle: Transition.noTransition, routeName: <String routeName>)`
it works like a charm.

**Reply by FilledStacks** [reply]
@rr-appadaptive Got it. I think we now have the ability to expose that functionality to normal navigation method. I'll look into this.
